lib/install.ps1

```diff
@@ -555,6 +555,8 @@ function dl_urls($app, $version, $manifest, $bucket, $architecture, $dir, $use_c
             }
         } elseif(Test-7zipRequirement -File $fname) { # 7zip
             $extract_fn = 'Expand-7zipArchive'
+        } elseif($fname -match '\.zst$') {
```

Comment:
Why do you create function when you do not use it then?

Reply:
> 
> 
> Why do you create function when you do not use it then?

it is used inside function install_deps($manifest, $arch) {

Should I remove the the function? What should I implement inside install_deps.

Reply:
What are you talking about?

You have function `Test-ZstdRequirement` exactly for this reason